cli/tauri.js/package.json

```diff
@@ -67,6 +67,7 @@
     "isbinaryfile": "4.0.6",
     "jsdom": "16.4.0",
     "lodash": "4.17.20",
+    "lookpath": "^1.1.0",
```

Comment:
Not sure if we should add another dep here (and if we do the version should be locked, not `^`). Instead, we could just exec `npm --version` and if that succeeds then we know npm is installed, otherwise assume it isn't. I think it might be better to add this as a util file instead of adding another dependency.